### PR TITLE
feat(compose): Show recently used recipients under input forms

### PR DESCRIPTION
**Pull request by tadzik**
TODO:

- [x] Tests
- [x] Fine-tune the amount of contacts suggested
- [x] Make layout more responsive (WIP in https://github.com/runbox/runbox7/pull/649)

### CODE REVIEW

**Comment by tadzik** [comment]
One suggested feature was to allow blacklisting of some recipients – but I think that'll require some API support to make it sync between devices: otherwise it may end up more frustrating than useful.

File: src/app/app.component.html [hunk 1]
```diff
@@ -35,6 +35,12 @@
       <mat-divider></mat-divider>
     </mat-nav-list>
 
+    <app-popular-recipients
+        (recipientClicked)="searchFor($event)"
```
Comment (castaway) [inline src/app/app.component.html:39]: Should this be "from:$event" ? Cos like this it finds mentions of the address inside other emails too

File: src/app/compose/compose.component.scss [hunk 2]
```diff
@@ -75,3 +63,87 @@ button {
...
+    }
+}
+
+.draft-buttons {
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    flex-grow: 1;
+
+    #trashDraft { order: 1 }
```
Comment (castaway) [inline src/app/compose/compose.component.scss:120]: is this part of a fix for a different issue?

Comment (castaway) [reply src/app/compose/compose.component.scss:120]: .. nm Im looking at the whole thing at once

File: src/app/compose/compose.component.html [hunk 3]
```diff
@@ -43,18 +39,55 @@
...
+        <div class="recipientSuggestionContainer" *ngIf="suggestedRecipients.length > 0 && editing">
+            <span>Recently used</span>
+            <div class="recipientSuggestionList">
+                <div *ngFor="let recipient of filteredSuggestions" class="suggestionItem">
+                    <mat-chip
+                        (click)="addRecipientFromSuggestions(recipient)"
+                        draggable="true"
+                        (dragstart)="recipientDragged($event, recipient)"
+                    >
+                        {{ recipient.nameAndAddress }}
```
Comment (castaway) [inline src/app/compose/compose.component.html:71]: I'm getting "null <address>" for those with no names :(

**Comment by castaway** [comment]
On my initial load, I got "Popular recipients" listed on the main nav menu, but opening compose I got no "recently Used", they appeared on a page reload. Not sure if thats just me,

Looks great else!

**Pull request merged**